main.rb

```diff
@@ -0,0 +1,11 @@
+require_relative 'my_list'
+
+list = MyList.new([1, 2, 3, 4])
+
+puts(list.all? { |e| e < 5 })
+puts(list.all? { |e| e > 5 })
+
+puts(list.any? { |e| e == 2 })
+puts(list.any? { |e| e == 5 })
+
+puts(list.filter(&:even?))
```

Comment:
- [ ] For the filter I am getting this error kindly fix it.
![Screenshot from 2023-07-21 01-38-11](https://github.com/astroboyReloaded/My_List_ruby/assets/26767383/d4bcb330-0e88-4f35-8932-c1601fd954fe)

Reply:
there is no error on my console.

Reply:
![image](https://github.com/astroboyReloaded/My_List_ruby/assets/128397949/567d05b0-437b-448b-832f-401fa81550e8)

@Deepakdanger

Reply:
All tests pass. Well resolved ✔️ 

![image](https://github.com/astroboyReloaded/My_List_ruby/assets/38088818/53bed59a-394f-4008-8ee1-d2749c09635e)

Reply:
@KDlamini i didnt change anything on filter method. i was a problem with past reviewer

Reply:
@KDlamini thanks